runtime/doc/options.txt

```diff
@@ -2387,14 +2387,18 @@ A jump table for the options with a short description can be found at |Q_op|.
 	  item		default		Used for ~
 	  stl:c		' ' or '^'	statusline of the current window
 	  stlnc:c	' ' or '='	statusline of the non-current windows
-	  vert:c	'│'		vertical separators |:vsplit|
-	  fold:c	'·'		filling 'foldtext'
+	  vert:c	'│' or '|'	vertical separators |:vsplit|
+	  fold:c	'·' or '-'		filling 'foldtext'
 	  diff:c	'-'		deleted lines of the 'diff' option
 
 	Any one that is omitted will fall back to the default.  For "stl" and
 	"stlnc" the space will be used when there is highlighting, '^' or '='
 	otherwise.
 
+	For "vert" and "fold", the new defaults '│' and '·' being of ambiguous
+	width for CJK languages, nvim will fallback to the old defaults '|' and
+	'-' when |ambiwidth| is set to "double".
```

Comment:
I thought we agreed not to do this. Did you find a problem when ambiwitdh is used?

Reply:
I tried quickly but any way I could think was too complex or brittle so I followed your suggestion. I fell back on reverting to old defaults when no vert/fold fillchar is specified. I believe the behavior is not perfect but reasonable: When ambiwdith=double, users who don't set fillchars won't benefit from the new defaults (the only problem is if they try another neovim on a server for instance, they might wonder why with a same neovim version they don't see the same fillchar defaults), if they set vert or fold, the usual behavior applies. Users with ambiwidth=single will see the new defaults and nothing else changes.
The TUI dictates the use of monocell characters (for now).

Reply:
Ok. But let's not over-specify it, we don't need to document this in excruciating detail. Or move it to the doc for 'ambiwidth'.